homeassistant/components/vicare/__init__.py

```diff
@@ -35,6 +36,18 @@
 _TOKEN_FILENAME = "vicare_token.save"
 
 
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    if entry.version == 1:
+        _LOGGER.debug("Migrating from version %s", entry.version)
+        entry.version = 2
+        hass.config_entries.async_update_entry(
+            entry, options={**entry.options, CONF_EXTENDED_API: False}
+        )
+        _LOGGER.debug("Migration to version %s successful", entry.version)
```

Comment:
Bump the minor version instead, here's an example https://github.com/home-assistant/core/blob/8bc9ae67b14c58cddf614a867eaeb5b81ccf04e7/homeassistant/components/homeassistant_sky_connect/__init__.py#L25-L64